### DISCUSSION TITLE
Visual Studio Code

### DISCUSSION THREAD

**Discussion by joshyates1980** [GitbookIO/community#210] [0. General bugs and questions] [answered]
### Ask a question, start a conversation, or make an annoucement

I forked the repo: https://github.com/addisonschultz/GitSync

which I found from the Gitbook tutorial: https://youtu.be/Fm5hYBsRSXo

to begin exploring Github and Gitbook features and integrations. 

Here you'll see some changes: https://github.com/joshyates1980/GitSync/commit/d2fc25e15d114e186ddacd41ea0e0aa5a5b39a53

My question to the community and Gitbook developers:

Have you found any benefit to sync and use VSCode with Gitbook (and Github) for book projects?

I did sync my VSCode to explore: https://joshyates.blob.core.windows.net/files/vscode.png

**Comment by addisonschultz** [comment, marked as answer]
Hey @joshyates1980! Great question 😄 The short answer is, you can use whatever platform (GitBook or VSCode [or other IDE]) you like best! 

GitSync allows you and your team to work in the platform of choice, while making sure updates will be saved across both sides - so nothing will go out of sync. 

If you want to use VS Code, you definitely can! 

GitSync will automatically manage any changes you push to main (or whatever branch you configure in the original GitSync configuration), and publish them live.

Hope this helps!
